src/status_im/ui/screens/chat/message/message_old.cljs

```diff
@@ -97,17 +98,33 @@
        current-public-key
        (partial style/quoted-message-author (and outgoing (not pinned)))
        (and outgoing (not pinned))]]
-     (if (and image
-              ;; Disabling images for public-chats
-              (not public?))
-       [fast-image/fast-image {:style  {:width            56
-                                        :height           56
-                                        :background-color :black
-                                        :border-radius    4}
-                               :source {:uri image}}]
-       [react/text {:style           (style/quoted-message-text (and outgoing (not pinned)))
-                    :number-of-lines 5}
-        (components.reply/get-quoted-text-with-mentions parsed-text)])]))
+     (cond
+       (and image (not public?)) ;; Disabling images for public-chats
+       [fast-image/fast-image
+        {:style  {:width            56
+                  :height           56
+                  :background-color :black
```

Comment:
Maybe you can use a quo color constant instead of `:black`

Reply:
That's not my code. This is showing as a change due to formatting changes by the IDE. To fix all the unnecessary formatting changes caused by the IDE, I opened a new [PR](https://github.com/status-im/status-mobile/pull/13808). Please follow on the new [PR](https://github.com/status-im/status-mobile/pull/13808).